libs/src/services/creatorNode/constants.js

```diff
@@ -1,3 +1,11 @@
 module.exports = {
-  CREATOR_NODE_SERVICE_NAME: 'creator-node'
+  CREATOR_NODE_SERVICE_NAME: 'creator-node',
+  DECISION_TREE_STATE: Object.freeze({
+    GET_ALL_SERVICES: 'GET ALL SERVICES',
+    FILTERED_TO_WHITELIST: 'FILTERED TO WHITELIST',
```

Comment:
let's be consistent w/ the capitalization of this vs. the other one. I thing sentence case is good b/c these values are meant for analytics dashboards